### DISCUSSION TITLE
agg_state数据类型hdfs load加载与insert加载结果不一致

### DISCUSSION THREAD

**Discussion by yaona1** [apache/doris#26495] [A - General / Q&A]
insert sql:
create table doris_test (a int,b int sum, agg_st_1 agg_state max_by(int ,int)) 
AGGREGATE KEY (a) DISTRIBUTED BY HASH(a) BUCKETS 1;

insert into doris_test values (1,2,max_by_state(1,2));
insert into doris_test values (2,2,max_by_state(2,2));
insert into doris_test values (1,3,max_by_state(1,3));

select max_by_merge(agg_st_1)  from doris_test  --结果为1

hdfs load sql:
create table doris_test_1 (a int,b int sum, agg_st_1 agg_state max_by(int ,int)) 
AGGREGATE KEY (a) DISTRIBUTED BY HASH(a) BUCKETS 1;

LOAD LABEL regression.test_1
(
 DATA INFILE("hdfs://hdfs/test.csv")
INTO TABLE doris_test_1 
 (a,b) 
set(agg_st_1=max_by_state(a,b))
)
WITH  hdfs (XXXX)

select max_by_merge(agg_st_1)  from doris_test_1  --结果为822083584

test.csv的数据为
1,2
2,2
1,3

**Comment by LemonLiTree** [comment]
We will fix it as soon as possible

**Comment by BiteTheDDDDt** [comment]
https://github.com/apache/doris/pull/26581
